projects/plugins/jetpack/extensions/plugins/publicize/hooks/use-select-social-media/index.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor';
+
+export default function useSelectSocialMedia( deps = [] ) {
+	return useSelect(
```

Comment:
I'm not sure how much of an issue this is, but this means that we'll end up reselecting the connections data when the message changes. Should we decouple those things? I doubt there's a big performance hit, but it would be worth checking.

Reply:
yeah... that would make sense. Should we create two hooks?

Reply:
Addressed here 06a431ec54f40feb4d6f0235d3f66cc32f555d89